### PR TITLE
Add known issue for 8.15.x memory increase.

### PR DESCRIPTION
**Pull request by cmacknz**
- Relates https://github.com/elastic/beats/issues/41355

The same as https://github.com/elastic/beats/pull/41379 but for agent instead.

### CODE REVIEW

File: docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc [hunk 1]
```diff
@@ -190,6 +261,27 @@ If you're using {agent} on Windows with any integration which makes use of the A
...
+
+See {beats} issue link:https://github.com/elastic/beats/issues/40705[#40705].
+
+*Impact* +
+
+Memory usage may be higher than in previous releases depending on the throughput of {agent}. A fix is planned for 8.15.4.
+
+- The worst memory increase is for low-throughput configs with large queues.
+- For users whose queues were already sized proportionate to their throughput, memory use is increased but only marginally.
+- User that are affected can mitigate it by lowering their queue size.
```
Comment (kilfoyle) [inline docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc:281]:
```suggestion
- Affected users can mitigate the higher memory usage by lowering their queue size.
```

File: docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc [hunk 2]
```diff
@@ -152,6 +202,27 @@ same package versions.
...
```
Comment (kilfoyle) [inline docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc:222]:
```suggestion
- Affected users can mitigate the higher memory usage by lowering their queue size.
```

File: docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc [hunk 3]
```diff
@@ -64,6 +89,31 @@ Review important information about the {fleet} and {agent} 8.15.3 release.
...
```
Comment (kilfoyle) [inline docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc:113]:
```suggestion
- Affected users can mitigate the higher memory usage by lowering their queue size.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! Just a small suggested fix.

Thanks a lot for adding this @cmacknz!

File: docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc [hunk 4]
```diff
@@ -31,6 +31,31 @@ Also see:
...
```
Comment (cmacknz) [inline docs/en/ingest-management/release-notes/release-notes-8.15.asciidoc:55]:
```suggestion
- Affected users can mitigate the higher memory usage by lowering their queue size.
```

**Review by pierrehilbert — APPROVED** [review]

**Comment by lucabelluccini** [comment]
Thanks @cmacknz, @pierrehilbert, @kilfoyle for the prompt action.

**Review by lucabelluccini — APPROVED** [review]

**Review by kilfoyle — APPROVED** [review]

**Comment by kilfoyle** [comment]
@cmacknz For this to show up in the 8.15 docs we'll need to backport into the `8.x`, `8.16` and `8.15` branches. I can look after that unless you explicitly didn't intend to merge into those.

**Comment by cmacknz** [comment]
Just forgot to add the label, thanks!

**Pull request merged**